### ISSUE TITLE
Client showing server version and features

### ISSUE THREAD

**Issue by l-mb** [nextcloud/android#7231]
### Is your feature request related to a problem? Please describe.

Specifically, I was trying to use the E2E encryption with the Nextcloud Android app (3.13.1 on Android 11), and I was struggling to find out why this did not work or where it might have been available.

As a customer/user of a hosted solution, I do not have access to the admin panel to check which features are enabled or even easily check which version the server is running from the client, and which major functions may or may not be active that translate to functions in the app not being available.

(I can see which apps are enabled when I log in to the web client under Settings/App Order, but that is a different thing.)

### Describe the solution you'd like

In the client, perhaps in `Settings` or a new adjacent `Server Info` section, it should show which version the client is connected to and those features of the app that are not enabled due to version mismatch or being disabled.

### Describe alternatives you've considered

I checked the web UI as a logged in user, but the information I was looking for was not available there either. (I thought the "About" section might help, but it doesn't have even the version number. Nor in the Settings section.)

**Comment by tobiasKaminsky** [comment]
Hm. I am not sure about this.
Rather than showing that e.g. Collabora is enabled and it somehow does not work, we should focus on having everything running without problems.

**Comment by l-mb** [comment]
> Rather than showing that e.g. Collabora is enabled and it somehow does not work, we should focus on having everything running without problems.

Obviously. But for me as a user of a hosted offering, trying to understand *why* something is not working is really quite useful. Case in point: I went nuts looking for the E2EE features because there's no indication in the app that the server didn't have that enabled.

I can tell whether an app isn't there by logging in via the web.

But the client doesn't even let me know which version of Nextcloud I'm connected to, so I don't even know which documentation to reference.

**Comment by tobiasKaminsky** [comment]
> Case in point: I went nuts looking for the E2EE features because there's no indication in the app that the server didn't have that enabled.

It should be like this: If you can mark an empty folder as encrypted or see a lock icon on an encrypted folder, you have e2e on server, else not.
Same for e.g. Collabora/OnlyOffice, if you click on such a file and it opens with Collabora/OnlyOffice, it is installed. (or if you click on + button).
There should be no info needed, if / what app on server is enabled.

I guess this is more due to the fact that you are on a shared hoster…

**Comment by l-mb** [comment]
That is a pretty unintuitive UX from my perspective as a user without admin privileges to check versions or enabled features.
